infra-as-code/bicep/orchestration/hubPeeredSpoke/README.md

```diff
@@ -62,43 +62,69 @@ In this example, the spoke resources will be deployed to the resource group spec
 > For the examples below we assume you have downloaded or cloned the Git repo as-is and are in the root of the repository as your selected directory in your terminal of choice.
 
 ### Azure CLI
+
 ```bash
 # For Azure global regions
-az deployment mg create \
-    --template-file infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep \
-    --parameters @infra-as-code/bicep/orchestration/hubPeeredSpoke/parameters/hubPeeredSpoke.parameters.all.json \
-    --location eastus \
-    --management-group-id alz
+
+$inputObject = @(
+  '--name',                ('alz-HubPeeredSpoke-{0}' -f (-join (Get-Date -Format 'yyyyMMddTHHMMssffffZ')[0..63])),
+  '--parameters',          '@infra-as-code/bicep/orchestration/hubPeeredSpoke/parameters/hubPeeredSpoke.parameters.all.json',
+  '--location',            'EastUS',
+  '--management-group-id', 'alz',
+  '--template-file',       "infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep"
+)
+
+dateYMD=$(date +%Y%m%dT%H%M%S%NZ)
+NAME="alz-HubPeeredSpoke-${dateYMD}"
+PARAMETERS="@infra-as-code/bicep/orchestration/hubPeeredSpoke/parameters/hubPeeredSpoke.parameters.all.json"
+LOCATION="eastus"
+MGID="alz"
+TEMPLATEFILE="infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep"
```

Comment:
Can we standardize the order of these so at least templatefile is then followed by parameters?